docker-compose.yml

```diff
@@ -22,6 +22,7 @@ services:
       KEYCLOAK_CLIENT_ID: ${CODEX_FEASIBILITY_BACKEND_KEYCLOAK_CLIENT_ID:-feasibility-gui}
       KEYCLOAK_ALLOWED_ROLE: ${CODEX_FEASIBILITY_BACKEND_KEYCLOAK_ALLOWED_ROLE:-CODEX_USER}
       ONTOLOGY_FILES_FOLDER_UI: ${CODEX_FEASIBILITY_BACKEND_ONTOLOGY_FILES_FOLDER:-/opt/codex-feasibility-backend/ontology}
+      ONTOLOGY_DB_MIGRATION_FOLDER: ${ONTOLOGY_DB_MIGRATION_FOLDER:-/opt/codex-feasibility-backend/ontology/migration}
```

Comment:
Although we want to get rid of any `codex` mentions, it would nonetheless be good to stay consistent for the time being.
Thus, keeping the `CODEX_FEASIBILITY_BACKEND_` prefix should be preferred.

```suggestion
      ONTOLOGY_DB_MIGRATION_FOLDER: ${CODEX_FEASIBILITY_BACKEND_ONTOLOGY_DB_MIGRATION_FOLDER:-/opt/codex-feasibility-backend/ontology/migration}
```